### ISSUE TITLE
Slow query

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#11]
```
The "select.sql" performed on an attached database is very slow - cca 30
seconds on my computer. We tried to execute ANALYZE multiple times; we
tried EXPLAIN but it seems all relevant columns have indices. We tried the
newest H2 1.0.67. Can you help us?
```

Original issue reported on code.google.com by `mvy.w...@gmail.com` on 26 Feb 2008 at 10:42

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I could reproduce this problem. It is fixed in the trunk and will be included 
in the
next release (in about 2 weeks). Now the query takes about 0.03 seconds instead 
of 30
seconds. The problem was a bug in the outer join handling that lead to 
exponential
runtime in respect to outer join tables (your query has quite many outer join 
tables).


```

Original comment by `thomas.t...@gmail.com` on 28 Feb 2008 at 7:06
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Thanks! I'll wait and try it when it's released.
```

Original comment by `mvy.w...@gmail.com` on 29 Feb 2008 at 8:30

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.68
```

Original comment by `thomas.t...@gmail.com` on 15 Mar 2008 at 11:18
- Changed state: **Fixed**
